### ISSUE TITLE
Upgrade Node JS from 16 to 18

### ISSUE THREAD

**Issue by abdullahwaheed** [openedx/frontend-lib-content-components#279]
**Abstract**
Node 16's support will end on [September 11, 2023](https://nodejs.org/en/blog/announcements/nodejs16-eol/). So we have to upgrade node JS to v18 before that.

**Extras**
Project should use `.nvmrc` to have a consistent node version
We have to update Readme accordingly.

[Upgrade Node JS from 16 to 18](https://github.com/openedx/platform-roadmap/issues/243)

**Comment by jesperhodge** [comment]
Great issue. Let's consider doing this at the same time we are upgrading node to v18 on projects that are using this library, for example, frontend-app-course-authoring. Just so we don't create any emergent problems by possibly having different node versions.

**Comment by abdullahwaheed** [comment]
yes we are planning to upgrade all the MFEs at once. Our team is creating draft PRs in all the repos and can be merged at the same time. e.g: #281 and https://github.com/openedx/frontend-app-course-authoring/pull/464
